style.css

```diff
@@ -1160,39 +1240,53 @@ body {
   }
 }
 
-@media only screen and (min-width: 1060px) and (max-width: 1290px) {
+@media only screen and (min-width: 1060px) and (max-width: 1440px) {
```

Comment:
My recommendation is to remove the max width from here.
And that should be all.
I am sure the next reviewer will approve the CRR.
But at the moment, when we open the app, we see directly the hamburger button on a desktop screen, when it should not be the case.

This line of code should look like this: 
`@media only screen and (min-width: 1060px) {`